src/lib/components/lines-and-dots/workflow-callback.svelte

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+  import Alert from '$lib/holocene/alert.svelte';
+  import CodeBlock from '$lib/holocene/code-block.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { timeFormat } from '$lib/stores/time-format';
+  import type { CallbackInfo } from '$lib/types';
+  import { formatDate } from '$lib/utilities/format-date';
+
+  export let callback: CallbackInfo;
+
+  $: completedTime = formatDate(callback.lastAttemptCompleteTime, $timeFormat);
+  $: nextTime = formatDate(callback.nextAttemptScheduleTime, $timeFormat);
+  $: failure = callback?.lastAttemptFailure?.message;
+
+  const titles = {
+    Standby: translate('nexus.callback.standby'),
+    Scheduled: translate('nexus.callback.scheduled'),
+    'Backing Off': translate('nexus.callback.backing-off'),
+    Failed: translate('nexus.callback.failed'),
+    Succeeded: translate('nexus.callback.succeeded'),
+  };
+
+  $: title = titles[callback.state] || translate('nexus.nexus-callback');
```

Comment:
Same comment here—I'd think that a referentially different `callback` property would trigger a re-render anyway?